builder/update.go

```diff
@@ -189,6 +189,9 @@ func (b *Builder) buildUpdateContent(params UpdateParameters, timer *stopWatch)
 		timer.Start("CREATE ZERO PACKS")
 		bundleDir := filepath.Join(b.Config.Builder.ServerStateDir, "image")
 		for _, bundle := range mom.Files {
+			if bundle.Type != swupd.TypeManifest {
+				continue
+			}
 			// TODO: Evaluate if it's worth using goroutines.
```

Comment:
this is NOT in the scope of this PR, but this really *is* worth using goroutines for. Need to evaluate if `swupd.CreatePack` is threadsafe, which I believe it is.